pkg/xds/generator/admin_proxy_generator.go

```diff
@@ -45,10 +47,15 @@ func (g AdminProxyGenerator) Generate(ctx context.Context, xdsCtx xds_context.Co
 	// In contrast to `AdminPort`, we shouldn't trust `AdminAddress` from the Envoy node metadata
 	// since it would allow a malicious user to manipulate that value and use Prometheus endpoint
```

Comment:
Is this comment no longer true/relevant?

Reply:
I re-read the comment and figured that we shouldn't allow other IPs except `127.0.0.1, ::1, 0.0.0.0, ::`, so slightly updated the comment and added validation

Reply:
yeah that makes complete sense actually :+1: